test/api/api_test.go

```diff
@@ -29,7 +29,7 @@ func TestUserTierCache(t *testing.T) {
 		t.SkipNow()
 	}
 	dbName := test.DBNameForTest(t.Name())
-	at, err := test.NewAccountsTester(dbName, nil)
+	at, err := test.NewAccountsTester(dbName, "", nil)
```

Comment:
nit: An approach that requires fewer line changes is having a `NewAccountsTesterCustom` that is used to add any new options that are necessary.  Then just the tests that need custom options have to be edited and all these default cases don't need to be touched. 

Something to consider for the future.

Reply:
Good advice. I will probably add that in a F/U.